Sources/Helpers/OAUtilities.m

```diff
@@ -565,6 +565,23 @@ + (UIImage *) imageWithColor:(UIColor *)color
     }
 }
 
++ (UIImage *) makeRoundedImage:(UIImage *)image radius:(CGFloat)radius
```

Comment:
yourImageView.layer.cornerRadius = yourRadius;
yourImageView.clipsToBounds = YES;

Delete this method